src/EnergyPlus/api/autosizing.cc

```diff
@@ -0,0 +1,102 @@
+//     software as "EnergyPlus version X" software, where "X" is the version number Licensee
+//     obtained under this License and may not use a different name for the software. Except as
+//     specifically required in this Section (4), Licensee shall not use in a company name, a
+//     product name, in advertising, publicity, or other promotional activities any name, trade
+//     name, trademark, logo, or other designation of "EnergyPlus", "E+", "e+" or confusingly
+//     similar designation, without the U.S. Department of Energy's prior written consent.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR
+// IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY
+// AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+// CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+// CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+// SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+// THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR
+// OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+// POSSIBILITY OF SUCH DAMAGE.
+
+#include <EnergyPlus/api/autosizing.h>
+#include <EnergyPlus/Autosizing/HeatingAirflowUASizing.hh>
+#include <EnergyPlus/DataSizing.hh>
+
+Sizer sizerHeatingAirflowUANew() {
+    auto sizer = new EnergyPlus::HeatingAirflowUASizer();
+    return reinterpret_cast<Sizer>(sizer);
+}
+void sizerHeatingAirflowUADelete(Sizer sizer) {
+    delete reinterpret_cast<EnergyPlus::HeatingAirflowUASizer *>(sizer);
+}
+void sizerHeatingAirflowUAInitializeForSingleDuctZoneTerminal(Sizer sizer, Real64 elevation, Real64 mainFlowRate) {
+    auto s = reinterpret_cast<EnergyPlus::HeatingAirflowUASizer *>(sizer);
+    s->initializeForSingleDuctZoneTerminal(elevation, mainFlowRate);
+}
+void sizerHeatingAirflowUAInitializeForZoneInductionUnit(Sizer sizer, Real64 elevation, Real64 mainFlowRate, Real64 reheatMultiplier) {
+    auto s = reinterpret_cast<EnergyPlus::HeatingAirflowUASizer *>(sizer);
+    s->initializeForZoneInductionUnit(elevation, mainFlowRate, reheatMultiplier);
+}
+void sizerHeatingAirflowUAInitializeForZoneFanCoil(Sizer sizer, Real64 elevation, Real64 designHeatVolumeFlowRate) {
+    auto s = reinterpret_cast<EnergyPlus::HeatingAirflowUASizer *>(sizer);
+    s->initializeForZoneFanCoil(elevation, designHeatVolumeFlowRate);
+}
+void sizerHeatingAirflowUAInitializeForSystemOutdoorAir(Sizer sizer, Real64 elevation, Real64 overallSystemMassFlowRate, int DOAS) {
+    auto s = reinterpret_cast<EnergyPlus::HeatingAirflowUASizer *>(sizer);
+    s->initializeForSystemOutdoorAir(elevation, overallSystemMassFlowRate, DOAS == 1); // TODO: Verify true/false 0/1 status here
```

Comment:
In C, we don't have bools, so we rely on an integer.  True is 1, we just need to make sure the C API clients understand that.